server.js

```diff
@@ -61,8 +61,9 @@ app.use('/', (req, res) => {
   alt.bootstrap(JSON.stringify(res.locals.data || {}));
   const iso = new Iso();
 
-  const blogAppUrl = (req.url).indexOf('books-music-dvds/recommendations/staff-picks') !== -1;
-  const routes = blogAppUrl ? appRoutes.client : appRoutes.server;
+  const isApiRoute = (req.url).indexOf(appConfig.baseApiUrl) !== -1;
```

Comment:
This should be `baseMonthUrl`. This logic will have to be updated now since there are two separate routes.

Reply:
We should test this once we start doing the reverse proxy. Right now I think I just see "Not Found" pages.